account_mass_reconcile/models/mass_reconcile_history.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# © 2012-2016 Camptocamp SA (Guewen Baconnier, Damien Crier, Matthieu Dietrich)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, api, fields, _
+
+
+class MassReconcileHistory(models.Model):
+    """ Store an history of the runs per profile
+    Each history stores the list of reconciliations done"""
+
+    _name = 'mass.reconcile.history'
+    _rec_name = 'mass_reconcile_id'
+    _order = 'date DESC'
+
+    @api.one
+    @api.depends('reconcile_ids')
+    def _get_reconcile_line_ids(self):
+        move_line_ids = []
+        for reconcile in self.reconcile_ids:
+            move_lines = reconcile.mapped('reconciled_line_ids')
+            move_line_ids.extend(move_lines.ids)
```

Comment:
It should be possible to use nested notation in the `mapped` 

    for rec in self:
        rec.reconcile_line_ids = rec.mapped(
            'reconcile_ids.reconciled_line_ids'
        ).ids